docs/docs-beta/docs/guides/configuring-assets.md

```diff
@@ -1,35 +1,49 @@
 ---
-title: Configuring assets using the Dagster UI
-sidebar_label: Configure asset runs
+title: Configuring assets
```

Comment:
This guide is on the light side right now and could use some additional detail (ex: required/optional fields, attaching metadata, different data structures, etc.) but I think that's a post-beta addition.